docker/demisto-sdk/README.md

```diff
@@ -0,0 +1,54 @@
+# Demisto-SDK docker
+
+Run demisto-sdk commands from within a docker.
+
+You can use this image to run validations locally on your work environment or inside your ci.
+
+## The Content Repository
+
+To use the Demisto-SDK you should have a content-like repository (or use the official [content repository](https://github.com/demisto/content)).
+You can generate one from this [template](https://github.com/demisto/content-external-template)
+
+## Mounts
+
+We use volume mount in our environment to run on local content repository.
```

Comment:
```suggestion
Demisto-sdk uses volume mounts, to run on local content repository.
```

end of the sentence is unclear

Reply:
It's not the demisto-sdk that uses the volume mounts.